force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,32 +181,34 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
-         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        // Initialize the sum to 0
+        Integer sum = 0;
+    
+        // Loop through all numbers from 1 up to intLimit (inclusive)
+        for (Integer num = 1; num <= intLimit; num++) {
+            // Add the current number to the sum
+            sum = sum + num;
+        }
+        return sum;
     }
-
     /**
      * Question 9
      * Generate String "Hello World!" three times, with each occurrence separated by a semicolon.
      * Example: returnHelloWorld() should return "Hello World!; Hello World!; Hello World!; "
      * @return The string "Hello World!; Hello World!; Hello World!; "
      */
     public static String returnHelloWorld() {
-        // Initialize the result String
-
-        // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
-         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        // Initialize the result string as an empty string
+        String helloWorld = '';
+    
+        // Use a for loop to append 'Hello World!;' three times
+        for (Integer i = 1; i <= 3; i++) {
```

Comment:
You tend to see most people starting iterator at 0. I think it is personal preference. You can do this if it makes more sense to you, just be aware if you are using examples from others, etc.